src/main/java/com/xiaomi/infra/pegasus/rpc/async/MetaSession.java

```diff
@@ -72,33 +73,50 @@ public MetaSession(
 
   public static error_types getMetaServiceError(client_operator metaQueryOp) {
     if (metaQueryOp.rpc_error.errno != error_types.ERR_OK) return metaQueryOp.rpc_error.errno;
-    query_cfg_operator op = (query_cfg_operator) metaQueryOp;
-    return op.get_response().getErr().errno;
+
+    if (metaQueryOp instanceof query_cfg_operator) {
+      return ((query_cfg_operator) (metaQueryOp)).get_response().getErr().errno;
+    } else if (metaQueryOp instanceof create_app_operator) {
+      return ((create_app_operator) (metaQueryOp)).get_response().getErr().errno;
+    } else {
```

Comment:
May you may add other meta rpc, better use `switch`

Reply:
`switch` only support basic types such as int, enum in java;
```
switch(metaQueryOp)
```
the above code is impossible;
could you teach me how to implement it?

Reply:
OK